.cloud_build/dart_pad.yaml

```diff
@@ -1,12 +1,7 @@
 steps:
   - name: gcr.io/$PROJECT_ID/flutter:stable
-    entrypoint: 'dart'
-    args: ['pub', 'get']
-    dir: pkgs/dart_pad
-  - name: gcr.io/$PROJECT_ID/flutter:stable
-    entrypoint: 'dart'
-    args: ['run', 'tool/grind.dart', 'build']
-    dir: pkgs/dart_pad
+    args: ['build', 'web']
+    dir: pkgs/sketch_pad
   - name: gcr.io/$PROJECT_ID/firebase
-    args: ['deploy', '--project=$PROJECT_ID', '--only=hosting']
-    dir: pkgs/dart_pad
+    args: ['deploy', '--project=$PROJECT_ID', '--only=dartpad']
```

Comment:
Isn't the `--only` option the type of deploy you're doing and should still be `hosting`?